stress/client/main.go

```diff
@@ -241,6 +242,7 @@ func performRPCs(gauge *gauge, conn *grpc.ClientConn, selector *weightedRandomTe
 			interop.DoCustomMetadata(client, grpc.WaitForReady(true))
```

Comment:
I don't think we want `WaitForReady` anymore.  There are probably hysterical raisins for why it was once there, but it can be removed now.  Or in a later PR; either way.

Reply:
Later PR... Same reasons as with moving to under `/interop`.